sno/working_copy/postgis_adapter.py

```diff
@@ -58,16 +57,13 @@ def v2_schema_to_postgis_spec(schema, v2_obj):
     Generate the SQL CREATE TABLE spec from a V2 object eg:
     'fid INTEGER, geom GEOMETRY(POINT,2136), desc VARCHAR(128), PRIMARY KEY(fid)'
     """
-    result = [
-        SQL("{} {}").format(Identifier(col.name), SQL(v2_type_to_pg_type(col, v2_obj)))
-        for col in schema
-    ]
+    result = [f'"{col.name}" {v2_type_to_pg_type(col, v2_obj)}' for col in schema]
```

Comment:
this change seems wrong - column identifiers aren't escaped any more (`Identifier`)

Reply:
Fixed